lib/jobs/ucs-service-profile-discovery.js

```diff
@@ -134,7 +134,7 @@ function UcsServiceProfileDiscoveryJobFactory(
                         .spread(function(newNode){
                             var relations = [{
                                 relationType: 'associatedTo',
-                                targets: [newNode.associatedServer]
+                                targets: [members.associatedServer]
```

Comment:
After `map`, the member is not yet be a plural, so change `members`  to `member` will make sense.